libraries/eosiolib/capi/eosio/privileged.h

```diff
@@ -123,6 +123,16 @@ void set_kv_parameters_packed( const char* data, uint32_t datalen );
 __attribute__((eosio_wasm_import))
 void preactivate_feature( const struct capi_checksum256* feature_digest );
 
+/**
+ * Submits a finalizer set change to Hotstuff
+ * 
+ * @param data - pointer finalizer_set object packed as bytes
+ * @param len  - size of packed finalazer_set object
+ * @pre `data` is a valid pointer to a range of memory at least `len` bytes long that contains packed abi_finalizer_set data
+ */
+__attribute__((eosio_wasm_import))
+void set_finalizers( const char* data, uint32_t len );
```

Comment:
Can this go in this header file? Doesn't this prevent anyone that compiles with this header file from setcode until the protocol `instant_finality` is activated.

Reply:
well, if import is not not used - it is not included into wasm. I checked the wasm of test contracts.